### PR TITLE
Dev/zliao/dios 1838 h265 srt depacketizer

### PR DESCRIPTION
**Pull request by murillo128**

### CODE REVIEW

File: nbproject/configurations.xml [hunk 1]
```diff
@@ -235,6 +235,14 @@
           <in>h264depacketizer.cpp</in>
           <in>h264encoder.cpp</in>
         </df>
+        <df name="h265">
```
Comment (murillo128) [inline nbproject/configurations.xml:238]: we can actually remove this file completelly

File: src/h264/h264depacketizer.cpp [hunk 2]
```diff
@@ -102,7 +102,7 @@ MediaFrame* H264Depacketizer::AddPayload(const BYTE* payload, DWORD payloadLen)
 	//Get nalu size
 	DWORD nalSize = payloadLen;
 
-	//Debug("-H264 [NAL:%d,type:%d,size:%d]\n", payload[0], nalUnitType, nalSize);
+	Debug("-H264 [NAL:%d,type:%d,size:%d]\n", payload[0], nalUnitType, nalSize);
```
Comment (murillo128) [inline src/h264/h264depacketizer.cpp:105]: revert this change

File: src/h265/H265Depacketizer.h [hunk 3]
```diff
@@ -3,6 +3,7 @@
 #define	H265DEPACKETIZER_H
 #include "rtp.h"
 #include "video.h"
+#include "HEVCdescriptor.h"
```
Comment (murillo128) [inline src/h265/H265Depacketizer.h:6]: Capitalize the "D" - `HEVCDescriptor.h` (and same with the .cpp)

File: src/h265/h265.h [hunk 4]
```diff
@@ -0,0 +1,465 @@
...
+		Debug("\tframe_crop_top_offset=%u\n",			frame_crop_top_offset);
+		Debug("\tframe_crop_bottom_offset=%u\n",		frame_crop_bottom_offset);
+		Debug("/]\n");
+	}
+private:
+	BYTE			vps_id = 0;
+	BYTE			max_sub_layers_minus1 =	0;
+	BYTE			ext_or_max_sub_layers_minus1 = 0;
+	BYTE			temporal_id_nesting_flag = 0;
+	H265ProfileTierLevel generalProfileTierLevel;
```
Comment (murillo128) [inline src/h265/h265.h:268]: let's keep case consistence, while we use camelCase in the project, I am not against using snake_case for keeping consistency with h265 spec. So, change `generalProfileTierLevel` to `general_profile>_tier_level` (or all the other members to camelCase)

**Comment by murillo128** [comment]
@mildsunrise tpal

**Comment by ZitaLiao** [comment]
> 

Sure. Would finish some basic testing first. Then would fix comment + clean up code to make a PR for the descriptor first.

**Comment by ZitaLiao** [comment]
> Sure. Would finish some basic testing first. Then would fix comment + clean up code to make a PR for the descriptor first.

File: src/h265/H265Depacketizer.h (hunk 3 above)
Comment (ZitaLiao) [reply src/h265/H265Depacketizer.h:6]: Fixed. Would push later

File: nbproject/configurations.xml (hunk 1 above)
Comment (ZitaLiao) [reply nbproject/configurations.xml]: fixed.

File: src/h264/h264depacketizer.cpp (hunk 2 above)
Comment (ZitaLiao) [reply src/h264/h264depacketizer.cpp:105]: fixed.

File: src/h265/h265.h (hunk 4 above)
Comment (ZitaLiao) [reply src/h265/h265.h]: fixed.

**Comment by ZitaLiao** [comment]
Made a new branch for the same topic.
